plugins/schedule/base/lib/model/om/BaseScheduleEvent.php

```diff
@@ -2349,8 +2349,16 @@ public function buildPkeyCriteria()
 			if ($this->isColumnModified(ScheduleEventPeer::CUSTOM_DATA))
 			{
 				if (!is_null($this->custom_data_md5))
+				{
 					$criteria->add(ScheduleEventPeer::CUSTOM_DATA, "MD5(cast(" . ScheduleEventPeer::CUSTOM_DATA . " as char character set latin1)) = '$this->custom_data_md5'", Criteria::CUSTOM);
 					//casting to latin char set to avoid mysql and php md5 difference
+					if (kDataCenterMgr::isMultiDc()) // if multi DC configuration don't check costume data on other DC
```

Comment:
this is not clear to me

Reply:
@inbal-ben-david this is something that was added when running propel-gen.